sites/aviationpros.com/server/graphql/fragments/content-list.js

```diff
@@ -44,6 +44,20 @@ fragment WebsiteContentListFragment on Content {
       }
     }
   }
+  ... on MediaGallery {
+    images {
```

Comment:
This shouldn’t be on content lists. This is also the wrong type. it should be ContentMediaGallery